.github/dependabot.yml

```diff
@@ -55,3 +55,11 @@ updates:
           - "Grpc.*"
     labels:
       - "area-codeflow"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
Should/could this be `/.github/workflows`?

Reply:
It runs recursively and I didn't see any need to bake that path in (?). We'll notice if it does something off in a PR.